### ISSUE TITLE
Simulation: Etched Sigil does not trigger all Hydra runes

### ISSUE THREAD

**Issue by Binkenstein** [d07RiV/d3planner#446]
When running a simulation with Etched Sigil equipped the no rune, arcane and blazing versions do not trigger in simulation when channelling. Have run through each rune and only Lightning, Frost and Mammoth runes trigger as expected.

**Comment by d07RiV** [comment]
Seems to work fine for me, like so: https://www.d3planner.com/894709225

However, keep in mind that I haven't updated the simulator for quite a while so it's a couple patches behind now. I'm not sure it's worth maintaining anymore as it's rarely used and not well suited for AOE scenarios which are the main focus now (as opposed to single target RG/elite DPS that was key back when I originally made it).

**Comment by Binkenstein** [comment]
Makes sense. I can do additional analysis via spreadsheets as long as the ability calculations are correct.
